connector/src/main/java/org/jboss/as/connector/subsystems/datasources/AbstractDataSourceService.java

```diff
@@ -530,6 +530,8 @@ protected ManagedConnectionFactory createMcf(org.jboss.jca.common.api.metadata.d
             }
             if (dataSourceConfig.getConnectionUrl() != null) {
                 managedConnectionFactory.setConnectionURL(dataSourceConfig.getConnectionUrl());
+            } else {
+                managedConnectionFactory.setConnectionURL("");
```

Comment:
Hack to bypass https://github.com/ironjacamar/ironjacamar/blob/be70d486d8e1a35aa13816113dacace3a47775d9/adapters/src/main/java/org/jboss/jca/adapters/jdbc/local/LocalManagedConnectionFactory.java#L105